test/libdnf/set/test_set.cpp

```diff
@@ -59,16 +59,25 @@ void SetTest::test_set_basics() {
     CPPUNIT_ASSERT(s1.empty());
 
     s1 = {1, 4, 6};
-    s1.add(2);
+    CPPUNIT_ASSERT(s1.add(2));
     CPPUNIT_ASSERT((s1 == libdnf::Set<int>{1, 2, 4, 6}));
 
     s1 = {1, 4, 6};
-    s1.add(4);
+    CPPUNIT_ASSERT(!s1.add(4));
     CPPUNIT_ASSERT((s1 == libdnf::Set<int>{1, 4, 6}));
 
     s1 = {1, 4, 6};
-    s1.remove(4);
+    CPPUNIT_ASSERT(s1.remove(4));
```

Comment:
Python set works here differently, but it is not a problem (). In python it returns None or raises an exception.